sunpy/net/vso/vso.py

```diff
@@ -158,19 +159,11 @@ def make(self, atype, **kwargs):
         obj = self.api.get_type(f"VSO:{atype}")
         return obj(**kwargs)
 
-    def search(self, *query, response_format=None):
```

Comment:
My plan wasn't to remove this keyword in this release, it was to keep it and swap the default value over, so it defaults to the new with the option of reverting to the old.

Reply:
Any reason?

Reply:
Just that this is one of the oldest parts of sunpy and I expect people are still relying on it. (Also I expected that database would need more hacking)

Reply:
Fine, I will remove my last commit.